docs/doxygen/overviews/eventhandling.h

```diff
@@ -723,19 +723,19 @@ typedef void (wxEvtHandler::*MyPlotEventFunction)(MyPlotEvent&);
 // remember that you don't need this at all if you only use Bind<>() and that
 // you can replace MyPlotEventHandler(func) with just &func unless you use a
 // really old compiler
-#define MY_EVT_PLOT_CLICK(id, func) \
-    wx__DECLARE_EVT1(MY_PLOT_CLICKED, id, MyPlotEventHandler(func))
+#define EVT_MY_PLOT_CLICKED(id, func) \
+    wx__DECLARE_EVT1(EVT_MY_PLOT_CLICKED, id, MyPlotEventHandler(func))
```

Comment:
You can't use the same name for the event type (integer constant) and for the macro. This is why we use `wxEVT_XXX` and `EVT_XXX` for them respectively.

Here `wx` prefix would be wrong, but I guess we could use `my` instead.

Reply:
I see, sorry. I do not use event tables so I am not really familiar with the macros but looking at it it is obvious (the code still somehow built without warnings and ran correctly in MSVS 2019).

TBH, I would rather not use `#define myEVT_MY_PLOT_CLICKED` as "MY" is already everywhere in this event-related names but I could not come with a better solution.

Reply:
Maybe `myEVT_PLOT_CLICKED`?

Reply:
Wouldn't that mean removing `My`/`MY` from both event type and `wxEvent`-derived class names? 

I have also just realized that `MY_EVENT` event type used with `wxCommandEvent` in the code example in "Using Existing Event Classes" does not have `EVT_` in it. So the other solution (not liking this one either TBH), would be to not use `EVT_` in the event type name and have it only in the macro name.

Sorry for wasting your time on such trivial stuff, I'm itching to just close this PR.

Reply:
Sorry, I really don't have time for this now because I suspect that if I start looking into this, I'd want to rewrite all of this text and, while important, I really should be doing other things right now.

I trust you to improve it, but ideally the examples here should come from (or be very similar to) the actual code used in e.g. `samples/event/event.cpp`, so that we could be reasonably sure they indeed compile/work.

Also, if you're changing this, please emphasize that `Bind()` is preferred and event tables are legacy. TIA!

Reply:
By writing " itching to just close this PR" I did not mean to hurry you up but just ignore the issue so no more of your precious time is wasted on this.

I did check the event sample but unfortunately the sample does not show how to create a new event class; if it did I would have followed in its steps...

Reply:
It would definitely be nice to show creating a new event class in this sample...